packages/http-canonicalization/src/codecs.ts

```diff
@@ -1,7 +1,6 @@
 import { getEncode, type MemberType, type Program, type Type } from "@typespec/compiler";
 import type { Typekit } from "@typespec/compiler/typekit";
 import { isHeader } from "@typespec/http";
-import type { HttpCanonicalization } from "./http-canonicalization-classes.js";
```

Comment:
The changes here are because we need to figure out what the codec is before we construct a canonicalization (because the codec is used to determine the mutation key for a type).